conf/app.routes

```diff
@@ -3,3 +3,4 @@
 POST        /file-upload/upload/envelopes/:envelopeId/files/:fileId                       uk.gov.hmrc.fileupload.controllers.FileUploadController.uploadWithRedirection(envelopeId: uk.gov.hmrc.fileupload.EnvelopeId, fileId: uk.gov.hmrc.fileupload.FileId, `redirect-success-url`: Option[String], `redirect-error-url`: Option[String])
 GET         /internal-file-upload/download/envelopes/:envelopeId/files/:fileId            uk.gov.hmrc.fileupload.controllers.FileDownloadController.download(envelopeId: uk.gov.hmrc.fileupload.EnvelopeId, fileId: uk.gov.hmrc.fileupload.FileId)
 GET         /internal-file-upload/quarantine/envelopes/:envelopeId/files/:fileId          uk.gov.hmrc.fileupload.controllers.FileDownloadController.illegalDownloadFromQuarantine(envelopeId: uk.gov.hmrc.fileupload.EnvelopeId, fileId: uk.gov.hmrc.fileupload.FileId)
+POST        /internal-file-upload/zip/envelopes/:envelopeId                               uk.gov.hmrc.fileupload.controllers.FileDownloadController.zip(envelopeId: uk.gov.hmrc.fileupload.EnvelopeId)
```

Comment:
might be overthinking things, but do we have to be be careful trusting envelopeId if its used in creating files. wondering if it were possible to trigger the endpoint with a envelopeId like "../../../../etc/passwd" it would be possible to write/overwrite somewhere the service isn't allowed to?

Reply:
It's only used as a part of a path with AWS S3 (and has a `.zip` suffix) - I don't think it's an issue. Note, it's not part of the locally generated temporary file.
It does make me wonder if envelopeId should have been modeled with a UUID though, it just a String, which is generated with a UUID if not provided.